pyproject.toml

```
@@ -7,36 +7,36 @@ authors = ["Torchbox Ltd"]
 [tool.poetry.dependencies]
 python = "^3.9"
 Django = "~3.2"
-wagtail = "~4.1"
+wagtail = "~4.2"
 psycopg2 = "^2.9.3"
 gunicorn = {version = "^20.1.0", optional = true}
 wagtail-django-recaptcha = "1.0"
-django-pattern-library = "^0.7.0"
+django-pattern-library = "^1.0.0"
 whitenoise = "^6.1.0"
 dj-database-url = "^0.5.0"
 django-redis = "^5.2.0"
-django-storages = {extras = ["boto3"], version = "^1.12.3"}
-django-permissions-policy = "^4.14.0"
+django-storages = {extras = ["boto3"], version = "^1.13.2"}
+django-permissions-policy = "^4.15.0"
 django-referrer-policy = "~1.0"
 django-csp = "^3.7"
 scout-apm = "^2.25.1"
 django-basic-auth-ip-whitelist = "^0.5"
 wagtail-accessibility = "^0.2.1"
 wagtailfontawesome = "^1.2.1"
-django-phonenumber-field = "^6.1.0"
+django-phonenumber-field = "^7.0.2"
 phonenumbers = "^8.12.48"
-wagtail-purge = {git = "https://github.com/nickmoreton/wagtail-purge", rev = "wagtail-4"}
-wagtail-webstories = {git = "https://github.com/nickmoreton/wagtail-webstories", rev = "allow-wagtail-4"}
+wagtail-purge = {git = "https://github.com/nickmoreton/wagtail-purge", rev = "support/wagtail42"}
+wagtail-webstories = {git = "https://github.com/nickmoreton/wagtail-webstories", rev = "support/wagtail42"}
 wagtail-markdown = "^0.10.0"
```

```suggestion
wagtail-markdown = {git = "https://github.com/nickmoreton/wagtail-markdown", rev = "support/wagtail42"}
```

I was seeing:
```
/venv/lib/python3.9/site-packages/wagtailmarkdown/blocks.py:4: RemovedInWagtail50Warning: Importing from wagtail.core.blocks is deprecated. Use wagtail.blocks instead. See https://docs.wagtail.org/en/stable/releases/3.0.html#changes-to-module-paths
  from wagtail.core.blocks import TextBlock
```
and there's a PR waiting for the maintainer to release

---

Ah. Thanks for spotting this. 

Unfortunately when trying with this branch, the markdown block still doesn't seem to work properly. I am seeing the rich text buttons, which shouldn't be there, and the FE page doesn't render my markdown text correctly, my markdown text is just rendered as a single plain text line.

Wonder how much we use this and if we can just remove it. Will check.

---

I've updated the template now to use the markdown filter, as well as use your branch. It should now work.